### ISSUE TITLE
Setting sculk blocks needs worldedit.anyblock.nbt permission

### ISSUE THREAD

**Issue by Schuwi** [IntellectualSites/FastAsyncWorldEdit#2051]
### Server Implementation

Paper

### Server Version

1.19.2

### Describe the bug

Trying to use any of "sculk_shrieker", "sculk_sensor" or "sculk_catalyst" as a block pattern e.g. in the //set command requires the "worldedit.anyblock.nbt" unlike any other block.
I am not sure whether this is intended behaviour. If it is could you please explain why?

### To Reproduce

1. Make sure you do not have the "worldedit.anyblock.nbt" permission but otherwise permissions to use worldedit.
2. Execute a command taking a block pattern such as //set with "sculk_shrieker", "sculk_sensor" or "sculk_catalyst" as pattern.
3. Observe the error message: "<FAWE> Invalid value for <pattern>(You are not allowed to use nbt. Lacking permission: worldedit.anyblock.nbt), acceptable values are any patterns".

### Expected behaviour

No additional permission should be required than with any other block. The command should succeed.

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/ad3e672959034a7593dea63bbe1768bf

### Fawe Version

FastAsyncWorldEdit version 2.5.1-SNAPSHOT-332;f8acaf5

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by NotMyFault** [comment]
> No additional permission should be required than with any other block.

I tend to disagree. Sculk blocks can have specific nbt data set and fall into the same protection category like other nbt-heavy blocks.

**Comment by Schuwi** [comment]
Thank you for your response.

I was misinformed. I did not know of any other block requiring this permission but I just found out that containers like chests and barrels also require this permission.

However I could not find any documentation on this permission and am not quite clear over what this permission is intended to achieve. Any block entities such as beacons, jukeboxes can have attached NBT data but those do not require the permission. Also the [documentation on block patterns](https://worldedit.enginehub.org/en/latest/usage/general/patterns/) does not mention a way to actually set any NBT data so I do not understand what this permission is protecting against.

Apart from that, according to the information I can find on the [Minecraft Wiki](https://minecraft.fandom.com/wiki), out of the sculk blocks only the Sculk Catalyst has special NBT data attached.
